grpc/dispatcher_test.go

```diff
@@ -121,30 +119,6 @@ func TestNewDispatcher(t *testing.T) {
 				protocol.GRPC,
 				errors.New("grpc dispatcher: missing config (endpoint/service/method)")),
 		},
-		{
-			name: "invalid endpoint",
-			dispatcherConfig: DispatcherConfig{
-				Service:  service,
-				Method:   method,
-				Endpoint: ":1",
-			},
-			expected: nil,
-			expectedErr: fiberError.NewFiberError(
-				protocol.GRPC,
-				errors.New("grpc dispatcher: unable to get reflection information, ensure server reflection is enable and config are correct")),
-		},
-		{
-			name: "invalid response",
-			dispatcherConfig: DispatcherConfig{
-				Service:  service,
-				Method:   "fake method",
-				Endpoint: fmt.Sprintf(":%d", port),
-			},
-			expected: nil,
-			expectedErr: fiberError.NewFiberError(
-				protocol.GRPC,
-				errors.New("grpc dispatcher: unable to fetch file descriptors, ensure config are correct")),
-		},
```

Comment:
these are no longer required as Fiber does not use server reflection to fetch proto anymore